### PR TITLE
Adding console app doc

### PR DESCRIPTION
**Pull request by johnmwright**
and updating related code. Resolving AspDotNet docs issues.

### CODE REVIEW

File: docs/AspDotNet.md [hunk 1]
```diff
@@ -70,17 +70,12 @@ If you want to customize the views (adding links, etc.) you can add JavaScript f
 Settings.Current.Render.JSIncludes.Add("/Content/errors.js");
 ```
 
-If you want to store some custom key/value style data with an exception, you can set up `Settings.GetCustomData`, for example:
+If you want to store some custom key/value style data with an exception, you can use `.AddLogData` extension method, for example:
```
Comment (NickCraver) [inline docs/AspDotNet.md]: This set of changes isn't right...all that stuff still works (we're using it on SO), what were you hitting problems with?

File: docs/AspDotNet.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Web.Config example pieces for an IIS 7.5 deployment:
 This is all optional (except `StackExchange.Exceptional.ExceptionalModule` if you want to capture unhandled MVC exceptions automatically), you can setup completely via code as well.  Examples:
 
 ```c#
-StackExchange.Exceptional.ErrorStore.Setup("My Application", new SQLErrorStore(_connectionString));
+Exceptional.Configure(settings => settings.DefaultStore = new SQLErrorStore(applicationName: "My Application", connectionString: _connectionString));
```
Comment (NickCraver) [inline docs/AspDotNet.md]: This should still work as it did, will dig.

File: docs/AspDotNet.md (hunk 1 above)
Comment (johnmwright) [reply docs/AspDotNet.md]: sorry, no issue -- I was just trying to keep the different guides somewhat consistent with each other. I push an update shortly with this reverted.

Comment (johnmwright) [reply docs/AspDotNet.md]: this is back to the original

File: docs/AspDotNet.md (hunk 2 above)
Comment (NickCraver) [reply docs/AspDotNet.md]: Nope you're 100% right here - intentional change. Looks good!

**Pull request merged**